src/components/Main/CustomDialog/CustomDialog.container.js

```diff
@@ -0,0 +1,13 @@
+import { connect } from 'react-redux'
+
+import { updateWarningStatus } from 'state/status/actions'
+
+import CustomDialog from './CustomDialog'
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
you can remove dispatch here
```js
const mapDispatchToProps = {
  updateWarningStatus
}
```

Reply:
i know, but it's done like that in every container... i will refactor all mappings in a new PR if it's okay

Reply:
🙆‍♂